### ISSUE TITLE
[Suggestion] Save searches

### ISSUE THREAD

**Issue by skirchner989** [mezz/JustEnoughItems#525]
As a feature suggestion it might be cool to be able to save searches for later quick reference. The saved searches could appear as buttons on the left side of the inventory screen and when clicked would automatically apply the text into the search box.

If implemented, future releases may add the ability to group and reorder the saved searches.

**Comment by mezz** [comment]
This is something I've been thinking about, it would be useful.
You can hit the up arrow key to scroll through previous searches already.
It would be nice to have a way to save a specific recipe to look up again immediately though.

**Comment by evieluvsrainbows** [comment]
Yeah, this would be an amazing feature to have. I would love to see this implemented :smile:

@mezz Make it happen :D (eventually)
